features/admin-file.feature

```diff
@@ -0,0 +1,22 @@
+@admin-file @issue-457
+  Feature: An admin user can edit and delete attribute in admin file page
+    As an admin user
+    I can add/edit/save/delete attribute
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+    When I go to "/adminFile/update/"
+    Then I should see "Login"
+
+  @wip @issue-457 @javascript
+  Scenario: Sign in as admin and visit admin file update page
+    Given I sign in as an admin
+    When I go to "/adminFile/update/id/88252"
+    Then I should see a button "New Attribute"
```

Comment:
Hi @rija, I am trying to pass a behat test on this scenario. It threw the following error:
![image](https://user-images.githubusercontent.com/64770635/93581788-31140500-f9d4-11ea-8e3e-65d69f3edff9.png)
It is strange that the same scenario is passing in @pli888 branch. Can you help have a look?

Reply:
@kencho51 I did a new checkout of your code and managed to get your test working:
```
Running acceptance tests
+ [[ dev == \d\e\v ]]
+ bin/behat --tags '@wip&&~@facebook&&~orcid' -v --stop-on-failure
@admin-file @issue-457
Feature: An admin user can edit and delete attribute in admin file page
  As an admin user
  I can add/edit/save/delete attribute

  Background:                                                 # features/admin-file.feature:6
    Given Gigadb web site is loaded with production-like data # DatasetViewContext::gigadbWebSiteIsLoadedWithProductionLikeData()
      │ Initializing the database with production_like.pgdmp... Terminating DB Backend... Restarting php container for deployment project...
      │ 
    When I go to "/adminFile/update/"                         # Behat\MinkExtension\Context\MinkContext::visit()
    Then I should see "Login"                                 # Behat\MinkExtension\Context\MinkContext::assertPageContainsText()

  @wip @issue-457 @javascript
  Scenario: Sign in as admin and visit admin file update page # features/admin-file.feature:12
    Given I sign in as an admin                               # GigadbWebsiteContext::iSignInAsAnAdmin()
    When I go to "/adminFile/update/id/88252"                 # Behat\MinkExtension\Context\MinkContext::visit()
    Then I should see a button "New Attribute"                # DatasetViewContext::iShouldSeeAButton()

1 scenario (1 passed)
6 steps (6 passed)
```